### PR TITLE
test: Reduce maximum line length to 200

### PR DESCRIPTION
**Pull request by martinpitt**
With that, our test/static-code is getting the same maximum length as cockpit-machines, and we can share the script.

 - Builds on top of #16944

### CODE REVIEW

File: test/verify/check-pages [hunk 1]
```diff
@@ -23,6 +23,9 @@ import os
 import parent
 from testlib import *
 
+RHEL_DOC_BASE = ("https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/"
```
Comment (allisonkarlitskaya) [inline test/verify/check-pages:26]: Maybe we can better just `#noqa` this?

File: test/verify/packagelib.py [hunk 2]
```diff
@@ -156,13 +158,15 @@ def createDeb(self, name, version, depends, postinst, install, content, arch, pr
...
-        cmd = '''mkdir -p /tmp/b/DEBIAN {repo}
-                 printf "Package: {name}\nVersion: {ver}\nPriority: optional\nSection: test\nMaintainer: foo\nDepends: {deps}\nArchitecture: {arch}\nDescription: dummy {name}\n{provides}\n" > /tmp/b/DEBIAN/control
-                 {post}
-                 touch /tmp/b/stamp-{name}-{ver}
-                 dpkg -b /tmp/b {deb}
-                 rm -r /tmp/b
-                 '''.format(name=name, ver=version, deps=depends, deb=deb, post=postinstcode, repo=self.repo_dir, arch=arch, provides=provides)
+        cmd = f'''mkdir -p /tmp/b/DEBIAN {self.repo_dir}
+                  printf "Package: {name}\nVersion: {version}\nPriority: optional\nSection: test
+Maintainer: foo\nDepends: {depends}\nArchitecture: {arch}
```
Comment (allisonkarlitskaya) [inline test/verify/packagelib.py:163]: I had earlier rewritten this in a very nice way.  Something like:

```
    printf "%s: %s\n" \
        Maintainer  'foo'
        Depends '...' \
        .... \
    > /tmp/...
```

which allows keeping everything neatly intended.

File: test/verify/storagelib.py [hunk 3]
```diff
@@ -445,7 +445,14 @@ def dialog_with_retry(self, trigger, values, expect):
         self.dialog_wait_close()
 
     def udisks_objects(self):
-        return json.loads(self.machine.execute("""python3 -c 'import dbus, json; print(json.dumps(dbus.SystemBus().call_blocking("org.freedesktop.UDisks2", "/org/freedesktop/UDisks2", "org.freedesktop.DBus.ObjectManager", "GetManagedObjects", "", [])))'"""))
+        return json.loads(self.machine.execute(
+            """python3 -c '
+import dbus, json
```
Comment (allisonkarlitskaya) [inline test/verify/storagelib.py:450]: Can you indent that?

File: test/verify/storagelib.py [hunk 4]
```diff
@@ -445,7 +445,14 @@ def dialog_with_retry(self, trigger, values, expect):
         self.dialog_wait_close()
 
     def udisks_objects(self):
-        return json.loads(self.machine.execute("""python3 -c 'import dbus, json; print(json.dumps(dbus.SystemBus().call_blocking("org.freedesktop.UDisks2", "/org/freedesktop/UDisks2", "org.freedesktop.DBus.ObjectManager", "GetManagedObjects", "", [])))'"""))
+        return json.loads(self.machine.execute(
+            """python3 -c '
```
Comment (allisonkarlitskaya) [inline test/verify/storagelib.py:449]: imho this belongs on the previous line

File: test/verify/check-pages (hunk 1 above)
Comment (martinpitt) [reply test/verify/check-pages:26]: I'm not a fan of `#noqa` to second-guess our policy.  However, without the line break the line is 141 characters, so it's actually not critical for *this* PR. And this conflicts with "breaking up URLs is evil", so I'll just re-join this for now.

File: test/verify/storagelib.py (hunk 3 above)
Comment (martinpitt) [reply test/verify/storagelib.py:450]: Does this work for you?
```py
    def udisks_objects(self):
        return json.loads(self.machine.execute(["python3", "-c", textwrap.dedent("""
            import dbus, json
            print(json.dumps(dbus.SystemBus().call_blocking(
                "org.freedesktop.UDisks2",
                "/org/freedesktop/UDisks2",
                "org.freedesktop.DBus.ObjectManager",
                "GetManagedObjects", "", [])))""")]))
```

We could alternatively also stop calling python completely, and use `busctl -j call` instead.

Comment (martinpitt) [reply test/verify/storagelib.py:450]: Pro: This looks much nicer:
```py
    def udisks_objects(self):
        reply = self.machine.execute(["busctl", "call", "--json=pretty",
                                      "org.freedesktop.UDisks2", "/org/freedesktop/UDisks2",
                                      "org.freedesktop.DBus.ObjectManager", "GetManagedObjects"])
        return json.loads(reply)["data"][0]
```

Con: RHEL 8's systemd is too old to have a `--json` option :crying_cat_face:

File: test/verify/packagelib.py (hunk 2 above)
Comment (martinpitt) [reply test/verify/packagelib.py]: I don't like this that much, as it requires a long list of `%s: %s` templates, and keep their number in sync with the fields. I used a slightly different form now with dedent() and m.write().

Comment (allisonkarlitskaya) [reply test/verify/packagelib.py]:
```
$ printf "%s: %s\n" a b c d e f g h
a: b
c: d
e: f
g: h
```

Comment (martinpitt) [reply test/verify/packagelib.py]: TIL! However, I honestly don't feel like cycling this again, unless you insist. IMHO dedent() is quite decent as well.

**Review by allisonkarlitskaya — APPROVED** [review]

File: test/verify/packagelib.py (hunk 2 above)
Comment (allisonkarlitskaya) [reply test/verify/packagelib.py]: Sure.  I agree that it looks fine now with dedent.  Maybe even better.

**Pull request merged**